modules/publication/jsx/publicationIndex.js

```diff
@@ -0,0 +1,129 @@
+import FilterForm from 'FilterForm';
+import {Tabs, TabPane} from 'Tabs';
+import PublicationUploadForm from './uploadForm.js';
+import formatColumn from './columnFormatter';
+
+class PublicationIndex extends React.Component {
+  constructor() {
+    super();
```

Comment:
I think the standards are evolving for how to write Reactified menu filters. Since this is going to the major branch, the PR for the media module would serve as a good basis for how to update this.

Reply:
this is going to minor, i can update this to use some of the newer conventions but i don't think i can use the recently developed components